components/movementsensor/gpsrtk/gpsrtk.go

```diff
@@ -222,13 +224,16 @@ func newRTKMovementSensor(
 		g.logger.Info("ntrip_baud using default baud rate 38400")
 	}
 
-	if g.writepath != "" {
+	if g.writepath == "" {
```

Comment:
check with Olivia for shared bugs addressed so we don't have merged conflicts, I think you both hit this one